### ISSUE TITLE
[UI] UI Test II standard panel has unreadable or almost unreadable text in all themes

### ISSUE THREAD

**Issue by jarena3** [massif-press/compcon#2976]
## Description
In UI Test II, standard panels have text that is virtually unreadable due to lack of color contrast between the background and text.

## Screenshots
Light theme example (GMS Light):
<img width="1264" height="777" alt="image" src="https://github.com/user-attachments/assets/6af96ad9-2199-44ff-9953-576275039991" />

Dark theme example (HORUS Terminal):
<img width="1317" height="796" alt="image" src="https://github.com/user-attachments/assets/df0993e1-8fbe-44c6-97ed-49d65ca18563" />

## Other Info
It's worse in some themes than others, but still noticeable in all of them. The error persists with all variants on the top panel.

Comp/con version: 3.0.0
OS: Windows 11
Browser: Firefox

_Originally posted by @sealtrowitz in https://github.com/massif-press/.github/discussions/35_
